.github/workflows/check.yml

```
@@ -0,0 +1,34 @@
+name: "Check"
```

（細かいですが）

このファイルは他のリポジトリに合わせて`test.yml`としておきたいかも。わかりやすいので。
```suggestion
name: "Test"
```